docs/source/Publications.rst

```diff
@@ -0,0 +1,8 @@
+Publications
+============
+
+*SyReC Synthesizer* is an academic software. Thus, many of its built-in algorithms have been published as scientific papers.
+
+If you use *SyReC Synthesizer* in your work, we would appreciate if you cited :cite:labelpar:`wille2010syrec`, :cite:labelpar:`wille2016syrec` and  :cite:labelpar:`wille2019towardsHDLsynthesis`.
```

Comment:
```suggestion
If you use the *SyReC Synthesizer* in your work, we would appreciate if you cited :cite:labelpar:`wille2010syrec`, :cite:labelpar:`wille2016syrec` and  :cite:labelpar:`wille2019towardsHDLsynthesis`.
```

Reply:
It's a bit bold to ask for the citation of 3 papers here. Once the software journal is out, we should replace this with the journal.

Reply:
Yes, lets do that.